virtual_undercloud.yml

```diff
@@ -19,14 +19,31 @@
     - name: clean previous vm
       shell: |
          source {{ infrared_dir }}/.venv/bin/activate
-         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url.rhel7 }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
       retries: 3
       ignore_errors: yes
+      when: osp_release == 13
+
+    - name: clean previous vm
+      shell: |
+         source {{ infrared_dir }}/.venv/bin/activate
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url.rhel8 }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True --cleanup True
+      retries: 3
+      ignore_errors: yes
+      when: osp_release > 13
 
     - name: create undercloud vm
       shell: |
          source {{ infrared_dir }}/.venv/bin/activate
-         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True -vvv 2>&1 | tee log_virsh1
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url.rhel7 }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True -vvv 2>&1 | tee log_virsh1
+      when: osp_release == 13
+
+    - name: create undercloud vm
+      shell: |
+         source {{ infrared_dir }}/.venv/bin/activate
+         infrared virsh --host-address 127.0.0.1 --host-key /root/.ssh/id_rsa --image-url '{{ vm_image_url.rhel8 }}' --topology-network virtual_uc_network --topology-nodes virtual_uc_topology:1 --host-memory-overcommit True -vvv 2>&1 | tee log_virsh1
```

Comment:
add a task  to create VM for OSP13 or below

Reply:
i wrote it ==13 which is for below, i'll correct it